indexer/app/lib/periodic_indexer.rb

```diff
@@ -278,72 +296,127 @@ def run_index_round
 
       checkpoints = []
 
-      did_something = false
```

Comment:
@cfitz - yeah, removing it and just committing every time seems harmless to me. I'd probably keep the did_something check though just in case all the empty http traffic to solr causes confusion ("I haven't updated any records in the last 24 hours but Solr has received 10,000 commits!").